### PR TITLE
improve(rebate) Gas rebate script should use daily average UMA price instead of current UMA price

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**

Each transaction's UMA to pay should use historical average UMA-ETH price for that day. This smoothes the amount of UMA to send and makes it non-reliant on the current UMA-ETH spot price

Tests also fixed

### CODE REVIEW

File: packages/core/scripts/gas-rebate/test.js [hunk 1]
```diff
@@ -9,17 +9,6 @@ contract("Gas Rebate: index.js", function() {
   const TEST_END_BLOCK = 10778455;
   const REBATE_LABEL = 9999;
 
-  describe("getHistoricalEthPrice", function() {
```
Comment (nicholaspai) [inline packages/core/scripts/gas-rebate/test.js:12]: These functions no longer exist

File: packages/core/scripts/gas-rebate/rebates/Rebate_1.json [hunk 2]
```diff
@@ -1,152 +1,223 @@
...
-    "0xe6662ec7fc7c84231b29f0f0da6321673d100854": 1.6218741904515916,
-    "0x8bd16de8938a2ad16794ac7e7502896f156e370c": 8.408517356682127,
-    "0x2f840532892f91811f99cdaec703f322a779cc66": 8.039390044761545,
-    "0x83b19521180d533dbae360be77f95d0da3aef98e": 11.086509471110434,
-    "0xa08aca4a5d1a6813dcae969df892581047cfcc30": 6.674638805759128
-  }
-}
+    "rebate": 1,
+    "fromBlock": 10752294,
+    "toBlock": 10869637,
```
Comment (daywiss) [inline packages/core/scripts/gas-rebate/rebates/Rebate_1.json:4]: this block changed, is that ok? also was this meant to regen this file?

Comment (nicholaspai) [reply packages/core/scripts/gas-rebate/rebates/Rebate_1.json:4]: yes this is fine, this Rebate_1.json file is unofficial

**Review by chrismaree — APPROVED** [review]
great change. seems fairer to refund based on average price.

**Review by daywiss — APPROVED** [review]

**Pull request merged**